crowbar_framework/Gemfile

```diff
@@ -38,15 +38,21 @@ gem "easy_diff", "~> 0.0.5"
 
 gem "ohai", "~> 6.24.2"
 gem "chef", "~> 10.32.2"
+gem "net-http-digest_auth", "~> 1.4"
 
 gem "mixlib-shellout", "~> 1.3.0",
     require: "mixlib/shellout"
 
 gem "activerecord-session_store", "~> 0.1.0",
     require: "activerecord/session_store"
 
-gem "mime-types", "~> 1.25.0",
+if ENV["CI"] && ENV["CI"] == "true"
```

Comment:
I just removed this part again: https://github.com/crowbar/crowbar-core/pull/699